app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getCategories } = require("./controllers/games-controller");
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+// error handling
+app.get("/*", (req, res) => {
+  res.status(404).send({ msg: "invalid path" });
+});
+
+app.use((err, req, res, next) => {
+  console.log(err, "unhandled error");
```

Comment:
Try to make sure you don't have any console.log()'s left in your code. Remove :)